flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -203,40 +205,40 @@ public void testPrimaryKeyFieldsAtBeginningOfSchema() {
     LocalDate dt = LocalDate.of(2022, 3, 1);
     try {
       sql(
-          "CREATE TABLE %s(data STRING NOT NULL, dt DATE NOT NULL, id INT, PRIMARY KEY(data,dt) NOT ENFORCED) "
-              + "PARTITIONED BY (data) WITH %s",
+          "CREATE TABLE %s(name STRING NOT NULL, dt DATE NOT NULL, id INT, PRIMARY KEY(name,dt) NOT ENFORCED) "
```

Comment:
It still confusing to use `name dt` as the primary key. We can put `id` at the begining and use it and use `id dt` as the primary key.

Reply:
Also, I'd like to ask @kbendick that if we use the following sql, this test will be very similar to `testUpsertAndQuery`.
`"CREATE TABLE %s(id INT NOT NULL, dt DATE NOT NULL, name STRING, PRIMARY KEY(id,dt) NOT ENFORCED) "`

Can we merge the two into one?

Anything that cuts down the UT time, it's worth it.

Reply:
Merging these two methods into one will make this method look long because I can't reuse the rows, which may bring out confusion. So I still wonder should I do this.